src/shogun2-core/src/main/java/de/terrestris/shogun2/util/naming/PhysicalNamingStrategyShogun2.java

```diff
@@ -94,16 +104,17 @@ private Identifier convertToLimitedLowerCase(JdbcEnvironment context, Identifier
 	 * @return The identifier length limit for the given context. null
 	 *         otherwise.
 	 */
-	private Integer getIdentifierLengthLimit(JdbcEnvironment context) {
-
+	protected Integer getIdentifierLengthLimit(JdbcEnvironment context) {
 		// https://docs.jboss.org/hibernate/orm/5.0/javadocs/org/hibernate/dialect/package-summary.html
 		String dialectName = context.getDialect().getClass().getSimpleName();
 
 		if (dialectName.startsWith("Oracle")) {
 			// identifier limit of 30 chars -->
 			// http://stackoverflow.com/a/756569
 			return LENGTH_LIMIT_ORACLE;
-
+		} if (context.getDialect() instanceof Shogun2OracleDialect) {
```

Comment:
I think this should be `else if` instead of `if`, right?

Reply:
Nice catch! Correct, fixed in f162d9ab49